.gitignore

```diff
@@ -79,3 +79,4 @@ tags
 /debug.sh
 /tmp/
 /reapply-all.sh
+
```

Comment:
Newline?

Reply:
lines should end in newlines / linebreaks according to [POSIX convention](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)